### PR TITLE
Meta jolie2java code generation

### PR DESCRIPTION
**Pull request by kicito**

### CODE REVIEW

File: packages/meta.ol [hunk 1]
```diff
@@ -42,8 +41,7 @@ type IntRefinement {
 }
 
 /// A long basic type.
-type LongBasicType {
-	long
+type LongBasicType: long {
```
Comment (fmontesi) [inline packages/meta.ol:44]: This and the other *BasicType are not mistakes. I actually need a subnode called 'long' or something like that. Maybe we can use an annotation to generate something with a different name in Java?

**Review by fmontesi — CHANGES_REQUESTED** [review]

File: packages/meta.ol (hunk 1 above)
Comment (kicito) [reply packages/meta.ol:44]: Java reserves void, int, enum, long, and double, so we'd need different names.

Comment (kicito) [reply packages/meta.ol:44]: would is... e.g. `isInt` make sense?

Comment (fmontesi) [reply packages/meta.ol:44]: Yes. :-)

**Pull request merged**